monai/bundle/scripts.py

```diff
@@ -580,102 +581,59 @@ def get_bundle_info(
-        tracking: if not None, enable the experiment tracking at runtime with optionally configurable and extensible.
-            if "mlflow", will add `MLFlowHandler` to the parsed bundle with default tracking settings,
-            if other string, treat it as file path to load the tracking settings.
-            if `dict`, treat it as tracking settings.
-            will patch the target config content with `tracking handlers` and the top-level items of `configs`.
-            for detailed usage examples, plesae check the tutorial:
-            https://github.com/Project-MONAI/tutorials/blob/main/experiment_management/bundle_integrate_mlflow.ipynb.
-        args_file: a JSON or YAML file to provide default values for `runner_id`, `meta_file`,
-            `config_file`, `logging`, and override pairs. so that the command line inputs can be simplified.
-        override: id-value pairs to override or add the corresponding config content.
-            e.g. ``--net#input_chns 42``.
+        workflow: specified bundle workflow name, should be a string or class, default to "ConfigWorkflow".
+        args_file: a JSON or YAML file to provide default values for this API.
+            so that the command line inputs can be simplified.
+        kwargs: arguments to instantiate the workflow class.
 
     """
 
-    _args = _update_args(
-        args=args_file,
-        run_id=run_id,
-        init_id=init_id,
-        final_id=final_id,
-        meta_file=meta_file,
-        config_file=config_file,
-        logging_file=logging_file,
-        tracking=tracking,
-        **override,
-    )
-    if "config_file" not in _args:
-        warnings.warn("`config_file` not provided for 'monai.bundle run'.")
+    _args = _update_args(args=args_file, workflow=workflow, **kwargs)
     _log_input_summary(tag="run", args=_args)
-    config_file_, meta_file_, init_id_, run_id_, final_id_, logging_file_, tracking_ = _pop_args(
-        _args,
-        config_file=None,
-        meta_file="configs/metadata.json",
-        init_id="initialize",
-        run_id="run",
-        final_id="finalize",
-        logging_file="configs/logging.conf",
-        tracking=None,
-    )
-    workflow = ConfigWorkflow(
-        config_file=config_file_,
-        meta_file=meta_file_,
-        logging_file=logging_file_,
-        init_id=init_id_,
-        run_id=run_id_,
-        final_id=final_id_,
-        tracking=tracking_,
-        **_args,
-    )
-    workflow.initialize()
-    workflow.run()
-    workflow.finalize()
+    (workflow_name,) = _pop_args(_args, workflow=ConfigWorkflow)
+    if isinstance(workflow_name, str):
+        workflow_class, has_built_in = optional_import("monai.bundle", name=f"{workflow_name}")  # search built-in
+        if not has_built_in:
+            workflow_class = locate(f"{workflow_name}")  # search dotted path
+        if workflow_class is None:
+            raise ValueError(f"can not locate specified workflow class: {workflow_name}.")
```

Comment:
```suggestion
            raise ValueError(f"Cannot locate specified workflow class: {workflow_name}.")
```

Reply:
updated in the latest commit.

Thanks.